tests/test_converters.py

```diff
@@ -110,7 +110,11 @@
                 ],
             },
         ),
-        (Guideline(x=0, name="foo"), {"x": 0, "name": "foo"}),
+        (Guideline(x=0, name="foo"), {"name": "foo"}),
```

Comment:
`{"name": "foo"}` doesn't say much about this guideline.. is it horizontal or vertical? it actually was vertical before, and now it turned into an implicitly horizontal one because angle (from horizontal axis) defaults to 0 now.

They should have really bumped the UFO spec minor version at least...